doc/changes/DM-34175.feature.rst

```diff
@@ -0,0 +1,7 @@
+* Made choice of `ObservationInfo` subclass and required properties configurable
```

Comment:
I don't think this is what is happening any longer given the current state of lsst/astro_metadata_translator#54